src/schema/v2/me/collection.ts

```diff
@@ -55,6 +111,9 @@ export const Collection: GraphQLFieldConfig<any, ResolverContext> = {
       private: true,
     })
 
-    return response
+    return {
+      ...response,
+      user_id: meID,
+    }
```

Comment:
Since `artworkConnection` is two hops away in the graph from `me`, i.e.

`me` → `collection` → `artworkConnection`

I needed a way to thread the `me` user_id down. Since the intervening collection response from Gravity doesn't contain it, I just inject it here, so that I can access it from inside the `artworkConnection` on L40 above. 

I _think_ this is idiomatic enough, but If there is a better way I'm all ears.